### ISSUE TITLE
Mybizna Installlation Issues

### ISSUE THREAD

**Issue by rashidtufail84** [mybizna/mybizna#3]
I am following this guide
https://mybizna.gitbook.io/mybizna-erp/readme/how-to-install-in-laravel/how-to-install-as-new-laravel-project

when i run the following command
php artisan vendor:publish --provider="Mybizna\Assets\Providers\MybiznaAssetsProvider"

it does not publish any assets or migration as stated in the documentation

secondly the following command is not  present in the system
php artisan automigrator:migrate

**Comment by YnievesDotNet** [comment]
ohh, in this moment is not functional? I try some way to put on, but, nothing

**Comment by dedanirungu** [comment]
Yes, it is not functional. We will release a functional filament system in the first quarter of next year.
